src/c++/perf_analyzer/genai-perf/genai_perf/main.py

```diff
@@ -147,6 +169,10 @@ def run():
 
 
 def main():
+    # Check if help is requested early
+    if any(arg in sys.argv for arg in ("--help", "-h")):
+        return 0
```

Comment:
Am I misunderstanding this? This checks if `--help`/`-h` was supplied and exits without printing the help message, right? Don't we want to print it?

Reply:
You're right. 🤦 I'll fix this. I think this was for ease at some point, and I missed the failure case. I was also relying on unit tests, but we don't use main in the unit tests.